sdk/confidentialledger/confidential-ledger-rest/test-resources.json

```diff
@@ -0,0 +1,92 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "ledgerName": {
+      "type": "string",
+      "metadata": {
+        "defaultValue": "0000-test-js",
+        "description": "Ledger Name"
+      },
+      "minLength": 3,
+      "maxLength": 24
+    },
+    "principalId": {
+      "type": "string",
+      "defaultValue": "ec667af1-0642-45f0-be8a-b76758a35dde",
```

Comment:
It looks like this corresponds to your Microsoft user account. Are you planning on removing this before merging and just testing the role assignment to our main test principal?

Reply:
Yes, exactly!